auacm/app/database.py

```diff
@@ -0,0 +1,11 @@
+'''Database handlers.'''
+from sqlalchemy import create_engine
+from sqlalchemy.orm import Session
+from sqlalchemy.ext.automap import automap_base
+
+# database setup
+Base = automap_base()
+engine = create_engine('mysql://acm@localhost/acm')
```

Comment:
We might want to abstract this out into Config.py in the event that someone's mysql server doesn't have the account "acm" set up with the ACM database.